includes/CLI/Plugin_Check_Command.php

```diff
@@ -249,7 +259,7 @@ static function ( $dirs ) use ( $excluded_files ) {
 			$file_results = $this->flatten_file_results( $file_errors, $file_warnings );
 
 			if ( '' !== $error_severity || '' !== $warning_severity ) {
-				$file_results = $this->get_filtered_results_by_severity( $file_results, intval( $error_severity ), intval( $warning_severity ) );
+				$file_results = $this->get_filtered_results_by_severity( $file_results, intval( $error_severity ), intval( $warning_severity ), $include_low_severity_errors, $include_low_severity_warnings );
```

Comment:
This new implementation should be done for warnings also. Around line 275.

Reply:
ok